### ISSUE TITLE
Editor feature: Object placement on colliders

### ISSUE THREAD

**Issue by BlackCurtainStudio** [stride3d/stride#2561]
**Is your feature request related to a problem? Please describe.**
When placing objects in a scene (dragging an asset into the scene view), it is natural that one would like to place an object, for example, on top of a table that has a collider. Currently, the Y position of the placed object seems to default to zero regardless.

**Describe the solution you'd like**
Object placement in the editor scene view should react to colliders. This means that when I place a model (drag into the scene view) on top of another model in the scene view that has a collider, it should land on top of that collider and not default to Y position zero.

**Comment by johang88** [comment]
We could probably use the picking buffer to resolve world position under the mouse cursor. This would allow it to work even if there is not collider on the entity.

**Comment by BlackCurtainStudio** [comment]
> We could probably use the picking buffer to resolve world position under the mouse cursor. This would allow it to work even if there is not collider on the entity.

Yes, this could probably work as well. I realized later that my original feature description would likely only work for Prefab assets that has colliders attached to them as well.
